### PR TITLE
Cleanup form sample code in demo

### PR DESCRIPTION
**Pull request by weakish**

### CODE REVIEW

File: demo.md [hunk 1]
```diff
@@ -316,13 +316,13 @@ Forms are useful for all kinds of things on webpages. Contact forms, newsletter
...
   <p><strong>This is just a test form. It doesn't do anything.</strong></p>
   <p>
-  <select>
-		<option selected="selected" value="1">Title</option>
-		<option value="2">Mr</option>
-    <option value="3">Miss</option>
-    <option value="4">Mrs</option>
-    <option value="5">Other</option>
-	</select>
+  <label for="title-select">Title</label>
```
Comment (weakish) [inline demo.md:319]: I think "Title" is a label, not one of the option values as Mr/Ms/Mrs/etc

File: demo.md [hunk 2]
```diff
@@ -348,12 +348,12 @@ Forms are useful for all kinds of things on webpages. Contact forms, newsletter
   </p>
   <p>
   <label for="cars">Choose a car:</label>
-<select name="cars" id="cars" multiple>
- <option value="volvo">Volvo</option>
- <option value="saab">Saab</option>
- <option value="opel">Opel</option>
- <option value="audi">Audi</option>
-</select>
+  <select name="cars" id="cars" multiple>
```
Comment (weakish) [inline demo.md:351]: align the code here so it is consistent with the code around

File: demo.md [hunk 3]
```diff
@@ -405,7 +405,15 @@ Forms are useful for all kinds of things on webpages. Contact forms, newsletter
   <label>Message</label>
   <textarea rows="6"></textarea>
   </p>
-
+  <p>
```
Comment (weakish) [inline demo.md:409]: Previously, this was missed in the code block.

**Comment by lkhrs** [comment]
I’m merging this because of the missing `p` tag.

**Comment by waldyrious** [comment]
> I’m merging this because of the missing `p` tag.

Just out of curiosity, do you disagree with the rest of the change?

**Comment by lkhrs** [comment]
> Just out of curiosity, do you disagree with the rest of the change?

I want to avoid spending time reviewing changes that are entirely about code formatting. I'll accept formatting changes if they also functionally improve something at the same time, like in this case with the missing markup.

I appreciate all contributions, but my time is limited and I have to prioritize effectively for the project.

**Pull request merged**